protocol/communities/community.go

```diff
@@ -31,6 +31,9 @@ import (
 
 const signatureLength = 65
 
+// GrantExpirationTime interval of 7 days
+var GrantExpirationTime = 168 * time.Hour
```

Comment:
> I remembered that we aim to avoid having clients with unconfirmed memberships in case the control node goes offline over a weekend.

week**end** not a week :sweat_smile:

Reply:
I clarified with John, he said 5-7 days would be a reasonable time. Can change to 5, wdyt?

Reply:
Ah, okay. I didn't know that. The downside of a longer period is that once someone is removed, they will still possess a 'false' proof of membership.

Reply:
I can remove grant when a user gets kicked. But patched client can still ignore this workaround

Reply:
Good idea, that would help.

Hopefully, there won't be many patched clients with this workaround. Even if there are, 7 days is the acceptable fallback.